### PR TITLE
Feat-742/Support-sitemap-config

### PR DESCRIPTION
**Pull request by lukman7788**
# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1.  Support read sitemap xml config (#742 )

## How to test  
1.  `./bin/run_dev -s ./src/components/config/__tests__/sitemap.xml`
2.  `./bin/run_dev --create-config -s ./src/components/config/__tests__/sitemap.xml` 
imageimageimage

### CODE REVIEW

**Review by nicnocquee — CHANGES_REQUESTED** [review]

**Comment by lukman7788** [comment]
@nicnocquee updated with multi language pages support
image
will generated to
image

**Review by nicnocquee — APPROVED** [review]
Please add in the CLI option docs

**Comment by lukman7788** [comment]
> Please add in the CLI option docs

@nicnocquee updated
Screen Shot 2022-07-08 at 11 35 22

File: src/index.ts [hunk 1]
```diff
@@ -130,7 +130,7 @@ class Monika extends Command {
 
     'create-config': Flags.boolean({
       description:
-        'Create config from HAR (-H), postman (-p), insomnia (-I) export file, or open Monika Configuration Generator using default browser',
+        'Create config from HAR (-H), postman (-p), insomnia (-I), sitemap (-s) export file, or open Monika Configuration Generator using default browser',
```
Comment (sapiderman) [inline src/index.ts:133]: -s is stun instead? is sitemape --sitemap?

**Review by sapiderman — CHANGES_REQUESTED** [review]
some fix for the --create-config docs and -s or --sitemap

File: src/index.ts (hunk 1 above)
Comment (lukman7788) [reply src/index.ts]: updated

**Review by sapiderman — APPROVED** [review]
all good
:metal: :v: :call_me_hand:

**Pull request merged**